prepare

```diff
@@ -73,6 +73,10 @@ for i in $(seq 1 90); do
     sleep 1s
 done
 
+# Show block devices and mountpoints
```

Comment:
Is this useful globally? Our test logs are already super-big, so it might not be worth adding additional clutter.

Reply:
Yeah this might be not the right place, I will tag you in a Slack thread for the context about what I am trying to do.